boards/arm/sensortile_box/sensortile_box.dts

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2019 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/l4/stm32l4r5Xi.dtsi>
+
+/ {
+	model = "STMicroelectronics SensorTile.box board";
+	compatible = "st,sensortile-box", "st,stm32l4r5";
+
+	chosen {
+		zephyr,console = &usart1;
+		zephyr,shell-uart = &usart1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		blue_led: led {
+			gpios = <&gpiob 15 GPIO_INT_ACTIVE_HIGH>;
+			label = "User LD1";
+		};
+
+		green_led: led_2 {
+			gpios = <&gpiof 2 GPIO_INT_ACTIVE_HIGH>;
+			label = "User LD4";
+		};
+	};
+
+	aliases {
+		led0 = &blue_led;
+		led1 = &green_led;
+	};
+};
+
+&usart1 {
+	current-speed = <115200>;
+	pinctrl-0 = <&usart1_pins_b>;
+	pinctrl-names = "default";
+	status = "okay";
+};
+
+&usart2 {
+	current-speed = <115200>;
+	pinctrl-0 = <&usart2_pins_a>;
+	pinctrl-names = "default";
+	status = "okay";
+};
+
+&i2c1 {
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_FAST>;
+
+	hts221@5f {
+		compatible = "st,hts221";
+		reg = <0x5f>;
+		label = "HTS221";
+		drdy-gpios = <&gpiod 13 GPIO_INT_ACTIVE_LOW>;
+	};
+
+	lps22hh@5d {
+		compatible = "st,lps22hh";
+		reg = <0x5d>;
+		label = "LPS22HH";
+		drdy-gpios = <&gpiod 15 GPIO_INT_ACTIVE_LOW>;
+	};
+};
+
+&i2c3 {
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_FAST>;
+
+	stts751@38 {
+		compatible = "st,stts751";
+		reg = <0x38>;
+		drdy-gpios = <&gpiog 5 GPIO_INT_ACTIVE_LOW>;
+		label = "STTS751";
+	};
+};
+
+&spi1 {
+	status = "okay";
+
+	cs-gpios = <&gpioe 11 0>, <&gpioe 12 0>;
+
+	lis2dw12@0 {
+		compatible = "st,lis2dw12";
+		spi-max-frequency = <1000000>;
+		reg = <0>;
+		irq-gpios = <&gpioc 5 0>;
+		label = "LIS2DW12";
+	};
+
+	lsm6dso@1 {
+		compatible = "st,lsm6dso";
+		spi-max-frequency = <1000000>;
+		reg = <1>;
+		irq-gpios = <&gpioa 2 0>;
+		label = "LSM6DSO";
+	};
+};
+
+&spi2 {
+	status = "okay";
```

Comment:
If you enable SPI2 in DT, better to do the same in board's Kconfig.defconfig.

Reply:
@ydamigos
Actually I do not need spi2. Shall I remove everything or deleting just the 'status' row?

Reply:
I removed everything